query/src/main/java/org/infinispan/query/impl/CacheQueryImpl.java

```diff
@@ -42,83 +36,36 @@
    private static final FetchOptions DEFAULT_FETCH_OPTIONS = new FetchOptions().fetchMode(FetchMode.EAGER);
 
    protected final AdvancedCache<?, ?> cache;
-   protected final KeyTransformationHandler keyTransformationHandler;
    protected final PartitionHandlingSupport partitionHandlingSupport;
    protected QueryDefinition queryDefinition;
-   private ProjectionConverter projectionConverter;
 
-   /**
-    * Create a CacheQueryImpl based on a Lucene query.
-    */
-   public CacheQueryImpl(Query luceneQuery, SearchIntegrator searchFactory, AdvancedCache<?, ?> cache,
-                         KeyTransformationHandler keyTransformationHandler, TimeoutExceptionFactory timeoutExceptionFactory,
-                         Class<?> entity) {
-      this(timeoutExceptionFactory == null ? searchFactory.createHSQuery(luceneQuery, entity) :
-                  searchFactory.createHSQuery(luceneQuery, entity).timeoutExceptionFactory(timeoutExceptionFactory),
-            cache, keyTransformationHandler);
-   }
+   // caching result size, since Search 6 doesn't do that
+   private Long resultSize;
```

Comment:
We are not caching query result size or list anymore:  https://infinispan.org/docs/stable/titles/upgrading/upgrading.html#querying 
Is this something you need to do?

Reply:
Right! There is no reason to `emulate` a deprecated behavior in a deprecated API. I'm removing it.